packages/e2e-tests/tests/regression-fe-1875-cypress-jump-to-code.test.ts

```diff
@@ -0,0 +1,36 @@
+import { openViewerTab, startTest } from "../helpers";
+import { waitForPaused } from "../helpers/pause-information-panel";
+import { getSelectedTestCase, getTestCaseSteps, openCypressTestPanel } from "../helpers/testsuites";
+import { waitForTimelineAdvanced } from "../helpers/timeline";
+import test, { expect } from "../testFixtureCloneRecording";
+
+// React Hook Form cypress/e2e/conditionalField.cy.ts
+// When this test is no longer supported, go to
+// https://app.replay.io/team/dzozMDAxY2NkMC1mNGNhLTQ1ZjctYTZkYy0zOGE2MjA2Y2U4M2I=/runs
+// and select the most recent cypress/e2e/conditionalField.cy.ts
+const recordingId = "122c4fe7-8a58-4c5c-9639-16b35b5f2ef1";
```

Comment:
Rather than referring to recording ids directly, tests should use an `exampleKey` (corresponding to `packages/e2e-tests/examples.json`) which should be passed to `test.use()` (see other tests for examples). This ensures recordings get cloned before running tests, to avoid two tests interfering with each other.